recipe/meta.yaml

```diff
@@ -12,7 +12,10 @@ build:
 
 requirements:
   run:
-    - conda-build
+    - conda-build         # [linux]
+    - conda-build 1.20.0  # [osx]
+    - conda-build 1.20.0  # [win64 and py34]
```

Comment:
We can probably inspect `%CONDA_PY%`, but then we are better off eliminating the post-link scripts. ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/issues/3 )